python/tvm/_ffi/_ctypes/node.py

```diff
@@ -60,4 +60,17 @@ def __getattr__(self, name):
                 "'%s' object has no attribute '%s'" % (str(type(self)), name))
         return RETURN_SWITCH[ret_type_code.value](ret_val)
 
+    def set_attr_value(self, name, value):
```

Comment:
Most of TVM's node is designed to be immutable, this means we need to avoid supporting set_attr_value deliberately. 

If we can build a node in one shot, we can likely use https://github.com/dmlc/tvm/blob/master/python/tvm/make.py#L86 instead.

Even if we want to support this function, we should set it to be ```_set_attr``` and clearly document that this function is not intended for normal use.